.werft/jobs/build/payment/stripe-config-secret.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+data:
+  settings: eyJwdWJsaXNoYWJsZUtleSI6InBrX3Rlc3RfNTFLeHVyN0dhZFJYbTUwbzNJNXJKQTNvbnkxdGNmdTNkM0NOd3BUWFR6QURkWTJISmlvRk1XTGdTa2M1d2h0UkZRam85UG5kM3pYYUdlcktQcXRmN0REQ3kwMFhBb01kbjZhIiwic2VjcmV0S2V5Ijoic2tfdGVzdF81MUt4dXI3R2FkUlhtNTBvM0NtVFJWc1Q2Q0xqd0VlSlhsWWtmdjZHajREQm42aVlVeDJQWUlUNDhjVlI5dlNUS0s1b2hwQTVCdWdycU5NUU9WVzN0NVJIODAwS011T3lEZ1QifQo=
```

Comment:
Should we really have a JWT encoded secret defined here? This is readable by anyone.

Reply:
Yes -- this is the "public" testing secret used for PR preview environments.

Reply:
Okay. Is this an isolated Stripe **account** which will not be used for production use? My primary concerns are that we should expect someone will use this key to access stripe. By doing so, they can extract test data but also burn through a rate limit. For this use-case, we should have an entirely isolated Stripe account such that there's no way for any of this data to show up while using a production account.

Reply:
Good point. There is no production Stripe account yet, but I agree it would make sense to create an entirely separate Stripe account for production when the time comes. Cc @geropl

Reply:
@mads-hartmann pointed out that we have a guide for adding secrets to Werft. IMO, it makes sense to follow that here https://www.notion.so/gitpod/Adding-a-secret-to-Werft-652d3cf3bd9d431e94da248901a64fe1.

If we do this, we should also burn the secret committed here.

Reply:
@easyCZ  Thx for checking here. The chargebee/Stripe "secrets" are a) for test accounts only, and b) added by intention; this was done to unblock us.

That being said, if we have a nicer way to do things (e.g., the Google Secret Manager), we should try to use it (in a follow-up PR).